### DISCUSSION TITLE
Seurat workflows and compatibility with targeted gene panels

### DISCUSSION THREAD

**Discussion by jhb1980** [satijalab/seurat#4288] [Q&A] [answered]
Hi Team (cc community)!

With BD and 10X Genomics offering off-the-shelf gene targeting panels for scRNA analysis (typically ~1,000 genes), I was wondering if you could comment / have experience with the suitability of the "default" Seurat framework (normalization / SCTransform, PCA, clustering etc) for working with these datasets...? 

Edit: see for example the public 10X neuroscience panel data on their [support site](https://support.10xgenomics.com/single-cell-gene-expression/datasets/4.0.0/Targeted_SC3v3_Human_Glioblastoma_Neuroscience). They detect a little over 1,100 (targeted) genes, but the median per-cell sits at 310 genes, presumably reflecting the differences in cell types. I'm not sure how many "housekeeping" genes are included in these panels...

Thanks!

**Comment by saketkc** [comment, marked as answer]
Hi @jhb1980,

Thanks for your question and thanks for bringing our attention to the newly released data! We will probably have a detailed vignette on these type of datasets soon, but I did some quick analysis below.

There are indeed some house keeping genes in the panel (and they exhibit the usual technical noise):
![image](https://user-images.githubusercontent.com/682153/113458896-e131e400-93e1-11eb-93df-0730903bae79.png)

SCTransform's variance stabilization works:
![image](https://user-images.githubusercontent.com/682153/113459017-4259b780-93e2-11eb-9385-088d31863c1a.png)

With most parameters as defaults, the UMAP:

```r
object <- SCTransform(object = object, method="glmGamPoi",  variable.features.n=500, verbose = FALSE)
object <- RunPCA(object = object verbose = FALSE)
object <- RunUMAP(object = object dims = 1:20, verbose = FALSE)
object <- FindNeighbors(object = object, dims = 1:20, verbose = FALSE)
object <- FindClusters(object = object, verbose = FALSE)
```

![image](https://user-images.githubusercontent.com/682153/113459314-1be84c00-93e3-11eb-9caf-2ee85ab6855a.png)
